### PR TITLE
Auth multitenancy support

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

**Comment by EricWittmann** [comment]
@carlesarnal can you close this and re-open it against the `multitenant-sr-demo` branch?

**Comment by famartinrh** [comment]
looks good!

File: app/src/main/java/io/apicurio/registry/services/tenant/CustomTenantConfigResolver.java [hunk 1]
```diff
@@ -27,49 +30,27 @@
...
-			final OidcTenantConfig config = new OidcTenantConfig();
-
-			config.setTenantId(tenantId);
-			config.setAuthServerUrl("http://localhost:8090/auth/realms/registry");
-			config.setClientId("tenant-a-api-client");
-			OidcTenantConfig.Credentials credentials = new OidcTenantConfig.Credentials();
-			config.setCredentials(credentials);
-
-			return config;
+		final TenantMetadataDto registryTenant = registryStorage.getTenantMetadata(tenantId);
```
Comment (EricWittmann) [inline app/src/main/java/io/apicurio/registry/services/tenant/CustomTenantConfigResolver.java:47]: What happens if the tenant metadata doesn't exist?

**Review by EricWittmann — COMMENTED** [review]
Nice work.

**Pull request merged**